Api/V8/OAuth2/Repository/AccessTokenRepository.php

```diff
@@ -84,7 +84,8 @@ public function persistNewAccessToken(AccessTokenEntityInterface $accessTokenEnt
     /**
      * @inheritdoc
      *
-     * @throws \InvalidArgumentException When access token is not found.
+     * @param $tokenId
+     * @throws \Exception
```

Comment:
🤔 this seems to remove useful information

Reply:
Yeah, auto-replace removed it when it replaced the invalid PHPDoc. I can potentially look at adding these back in but at least having valid doc blocks is a step in the right direction imo.